### PR TITLE
Update PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by yvovandoorn**
Be specific on making sure the default text is removed and what should be put in the body portion of the PR.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -1,3 +1,8 @@
-*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*
+*Please read take into consideration of the following:*
```
Comment (mattstratton) [inline .github/PULL_REQUEST_TEMPLATE:1]:
```suggestion
*Please read and take into consideration the following:*
```

**Review by mattstratton — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+- Title your pull request in this format: The event name and year in the title of the PR,
+- Describe what is being changed, i.e., `[CHI-2024] Add Bluth Company as a sponsor`
+- Remove this text and replace with a high level of the changes (e.g. 'Added 10 new speakers, Added 6 sponsors')
 
-If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.
+* If you are adding or removing organi[s/z]ers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.*
```
Comment (phrawzty) [inline .github/PULL_REQUEST_TEMPLATE:6]: While I _love_ messing around with this sort of thing as a lark<sup>1</sup>, for better or worse, I think the standard is `en_US` spelling, no?

<sup>1</sup> `The first DevOpsDays was held in G[h]<a|e>n<d|t>`

File: .github/PULL_REQUEST_TEMPLATE [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+
+Finally, if you left this text in your PR, then it appears you didn't read it. Please remove this in the PR you are submitting.
```
Comment (bridgetkromhout) [inline .github/PULL_REQUEST_TEMPLATE:8]: Do we want to give the appearance of unkindness? Perhaps just stating "Please ensure you have replaced this default text" would suffice?

Comment (phrawzty) [reply .github/PULL_REQUEST_TEMPLATE:8]: Yeah there's no reason to get aggro 😆

Comment (mattstratton) [reply .github/PULL_REQUEST_TEMPLATE:8]: I went back and forth on this one; i read it as passive aggressive at first, and now I think it is more in the spirit of "this is why we say to remove the text", but impact > intent so probably agree with Bridget

Comment (mattstratton) [reply .github/PULL_REQUEST_TEMPLATE:8]: that said, we have said "please remove this text" for years and nobody does it, which is why it got aggro here :D

Comment (mattstratton) [reply .github/PULL_REQUEST_TEMPLATE:8]: one other solution (which might not work) is we can probably write some kind of PR linting rule that will block the PR if the default text was not removed (it'll be pretty easy to match on something like "Bluth Company" which won't be in any legit PRs)

Comment (mattstratton) [reply .github/PULL_REQUEST_TEMPLATE:8]: https://github.com/orgs/community/discussions/84771

**Comment by mattstratton** [comment]
I actually put in a PR that will test the pull requests against default text being included (it just tests for the word `"Bluth"` but that should suffice) and that seems to be working (not yet merged)

https://github.com/devopsdays/devopsdays-web/pull/13643

File: .github/PULL_REQUEST_TEMPLATE (hunk 3 above)
Comment (mattstratton) [reply .github/PULL_REQUEST_TEMPLATE:8]: okay that was easy https://github.com/devopsdays/devopsdays-web/pull/13643

Comment (mattstratton) [reply .github/PULL_REQUEST_TEMPLATE:8]: i think this can probably be softened up now that #13643 has been merged

Comment (yvovandoorn) [reply .github/PULL_REQUEST_TEMPLATE:8]: The intent was not to be passive-aggressive. It is hard to come up with words on how to tell folks (kindly) to remove the default text

**Comment by yvovandoorn** [comment]
I updated the message to be much less aggro and far more approachable!

**Review by glasnt — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE [hunk 4]
```diff
@@ -1,3 +1,11 @@
-*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*
+*Please read and take into consideration the following:*
+- Title your pull request in this format: The event name and year in the title of the PR.
+- Describe what is being changed, i.e., `[CHI-2024] Add Bluth Company as a sponsor`.
+- Remove this text and replace with a high level of the changes (e.g. 'Added 10 new speakers, Added 6 sponsors').
```
Comment (phrawzty) [inline .github/PULL_REQUEST_TEMPLATE:4]:
```suggestion
- Title your pull request in this format: `[PLACE-YEAR] Brief description of the proposed change`. For example: `[CHI-2024] Add Bluth Company as a sponsor`
- Remove _this default text_ from the PR description, and replace it with a summary of the changes (e.g. 'Added 10 new speakers, Added 6 sponsors, Added emojis to CFP page').
```

**Review by phrawzty — CHANGES_REQUESTED** [review]

**Review by phrawzty — APPROVED** [review]

**Pull request merged**